.rubocop.yml

```
@@ -10,7 +10,7 @@ AllCops:
   Exclude:
     - bin/bundle
     - db/schema.rb
-    - db/migrate
+    - db/migrate/*
```

```suggestion
    - db/*
```
And it will allows you to remove the line before: `db/schema.rb` because this will skip all files inside `db/` folder